docs/Software Architecture Document/Components View/index.md

```diff
@@ -148,6 +148,21 @@ The view is represented by module and subsystem diagrams that show the system's
     - Not work for Android apps.
     - Not work for apps connected via SDL protocol version lower than 4.
 
+#### Plugin Manager
```

Comment:
Is it Global or `Commands` Plugin Manager?
Seems like it's a specific plugin manager with Commands only related functionality